xml/decoders/Digitrax_Economy_Series7.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet type="text/xsl" href="../XSLT/decoder.xsl"?>
+<!-- Copyright (C) JMRI 2002, 2005, 2007, 2014, 2016 All rights reserved -->
+<!--                                                                        -->
+<!-- JMRI is free software; you can redistribute it and/or modify it under  -->
+<!-- the terms of version 2 of the GNU General Public License as published  -->
+<!-- by the Free Software Foundation. See the "COPYING" file for a copy     -->
+<!-- of this license.                                                       -->
+<!--                                                                        -->
+<!-- JMRI is distributed in the hope that it will be useful, but WITHOUT    -->
+<!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
+<!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
+<!-- for more details.                                                      -->
+<decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
+  <version author="Peter Ulvestad" version="1" lastUpdated="20241008"/>
+
+  <!-- Created from Economy Series 6 -->
+  <!-- Version 0 - New Series 7 Economy Digitrax decoders -->
+  <decoder>
+    <family name="Economy Series 7 with FX3, silent" mfg="Digitrax" lowVersionID="204" highVersionID="204" comment="">
+      <model model="DN147A0" numOuts="2" numFns="5" maxMotorCurrent="1A (peak=2A)" formFactor="HO" connector="9pin" comment="">
+      </model>
+      <model model="DN147A1" numOuts="2" numFns="5" maxMotorCurrent="1A (peak=2A)" formFactor="HO" connector="9pin" comment="">
+      </model>
+      <model model="DN147K0A" numOuts="2" numFns="5" maxMotorCurrent="1A (peak=2A)" formFactor="HO" connector="9pin" comment="">
+      </model>
+      <model model="DN147K0B" numOuts="2" numFns="5" maxMotorCurrent="1A (peak=2A)" formFactor="HO" connector="9pin" comment="">
+      </model>
+      <model model="DN147K0D" numOuts="2" numFns="5" maxMotorCurrent="1A (peak=2A)" formFactor="HO" connector="9pin" comment="">
+      </model>
+      <model model="DN147K0E" numOuts="2" numFns="5" maxMotorCurrent="1A (peak=2A)" formFactor="HO" connector="9pin" comment="">
+      </model>
+      <model model="DN147K1C" numOuts="2" numFns="5" maxMotorCurrent="1A (peak=2A)" formFactor="HO" connector="9pin" comment="">
```

Comment:
The documentation says that this is a N-scale decoder. I don't know if that matters. But it seems to be designed for Kato N-locomotives.
https://www.digitrax.com/media/apps/products/mobile-decoders/dn147k1c/documents/DN147K1C.pdf

Reply:
You are correct and I missed changing that, there is one other correction I need to make

Reply:
File updated